tooling/src/hypothesistooling/__main__.py

```diff
@@ -477,7 +477,7 @@ def check_examples2():
 
 @examples_task
 def check_examples3():
-    run_tox("examples2", PY36)
+    run_tox("examples3", PY36)
```

Comment:
While we're here, is it worth bumping this to the Python 3.7?

(Or rather, having a constant `PY3_LATEST` which gets changed every time we bump the newest version of Python 3?)

Reply:
I thought about it, but decided to go for a minimal and obviously correct change. (the point-eight is to exactly match the pipelines 3.6 version)

Planning to revisit that once 3.8 is released 😉